cfme/fixtures/appliance.py

```diff
@@ -112,6 +112,15 @@ def temp_appliance_preconfig_funcscope_upgrade(appliance, pytestconfig):
         yield appliances[0]
 
 
+@pytest.fixture(scope="module")
+def temp_appliance_preconfig_long(appliance, pytestconfig):
+    """ temp appliance with 24h lease for auth tests """
+    with sprout_appliances(
+            appliance, config=pytestconfig, preconfigured=True, lease_time=1440
```

Comment:
:+1:

Reply:
@john-dupuy for auth tests, you don't want appliance on openstack correct? Can you specify just rhev? via `provider_type = rhevm`